test/generate-tests-utils.xspec

```diff
@@ -1,77 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- ===================================================================== -->
-<!--  File:       test/generate-tests-utils.xspec                          -->
-<!--  Author:     Florent Georges                                          -->
-<!--  Tags:                                                                -->
-<!--    Copyright (c) 2010 Jeni Tennsion (see end of file.)                -->
-<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
-<t:description xmlns:t="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:test="http://www.jenitennison.com/xslt/unit-test" query="http://www.jenitennison.com/xslt/unit-test" query-at="../src/compiler/generate-query-utils.xql" stylesheet="../src/compiler/generate-tests-utils.xsl">
+<t:description xmlns:t="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:test="http://www.jenitennison.com/xslt/unit-test" stylesheet="../src/compiler/generate-tests-utils.xsl">
```

Comment:
Good point.

But, it may not be appropriate to simply reinstate the original copyright description. Neither Florent nor Jeni wrote the code in this file. In this pull request, the original code has been migrated to [`generate-x-utils.xspec`](https://github.com/xspec/xspec/pull/83/files#diff-97ff03026f55d6b39c3c2e377a057ac4).
So this file is essentially derived from 86e9bbf676d027e14234338224977dc75b65d6b1 and 9b94529e489faf784716f66b15434d381ebcb423. What to do in this case?